src/config/2021.json

```diff
@@ -150,29 +150,32 @@
         {
           "part": "IV",
           "chapter_number": "18",
-          "title": "CDN",
-          "slug": "cdn",
-          "todo": "true"
+          "title": "Caching",
+          "slug": "caching"
```

Comment:
Could you revert any non-contributor changes to this file?

Reply:
I believe @tunetheweb did those. Do you want me to revert them anyway?

Reply:
I did indeed do those.

I wanted to be able to generate it easily when on this branch. Plus I wanted our CI to generate it, so we can see of any issues in generated HTML or Lighthouse.

Leave them as is and will revert before merging.